cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -157,7 +157,8 @@ func isListResource(msg *beehiveModel.Message) bool {
 	msgResource := msg.GetResource()
 	if strings.Contains(msgResource, beehiveModel.ResourceTypePodlist) ||
 		strings.Contains(msgResource, "membership") ||
-		strings.Contains(msgResource, "twin/cloud_updated") {
+		strings.Contains(msgResource, "twin/cloud_updated") ||
+		strings.Contains(msgResource, commonconst.ResourceTypeServiceAccount) {
```

Comment:
The `token request` is a k8s object, so it should not be the `List` resource here.

Reply:
but it seems to not be able to be distinguished from other model messages.

Reply:
Just as a k8s object like pod, cm, etc.

Reply:
Note: The tokenreview obj's resource version is empty, so it's reasonable to use the `List` here.